shopinvader_api_customer/models/res_partner.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2024 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class ResPartner(models.Model):
+
+    _inherit = "res.partner"
+
+    def _update_shopinvader_customer(self, values: dict) -> "ResPartner":
```

Comment:
We usually declare helper abstract models and make our routes depend on them. You can take inspiration from the whishlist api: https://github.com/shopinvader/odoo-shopinvader/blob/16.0/shopinvader_api_wishlist/dependencies.py